modules/multi-architecture-creating-x86_64-bootimage.adoc

```diff
@@ -0,0 +1,137 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-creating-x86-64-bootimage_{context}"]
+= Creating a 64-bit x86 boot image using the Azure image gallery
+
+The following procedure describes how to manually generate a 64-bit x86 boot image.
+
+.Prerequisites
+
+* You installed the Azure CLI (`az`).
+* You created a single-architecture Azure installer-provisioned cluster with the multi-architecture installer binary.
+
+.Procedure
+
+. Log in to your Azure account by running the following command:
++
+[source,terminal]
+----
+$ az login
+----
+. Create a storage account and upload the `x86_64` virtual hard disk (VHD) to your storage account. The {product-title} installation program creates a resource group. However, the boot image can also be uploaded to a custom named resource group:
++
+[source,terminal]
+----
+$ az storage account create -n ${STORAGE_ACCOUNT_NAME} -g ${RESOURCE_GROUP} -l westus --sku Standard_LRS <1>
+----
++
+<1> The `westus` object is an example region.
++
+. Create a storage container using the storage account you generated by running the following command:
++
+[source,terminal]
++
+----
+$ az storage container create -n ${CONTAINER_NAME} --account-name ${STORAGE_ACCOUNT_NAME}
+----
+. You must use the {product-title} installation program JSON file to extract the URL and `x86_64` VHD name:
```

Comment:
The first sentence of each step needs to be in the imperative mood [per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps):
```suggestion
. Use the {product-title} installation program JSON file to extract the URL and `x86_64` VHD name:
```

Reply:
Updated as per the suggestion.